tests/client-units/account_store.coffee

```diff
@@ -65,15 +65,15 @@ describe 'AccountStore initialized without account', ->
     it 'Then the default account is still the same', ->
         AccountStore.getDefault().get('id').should.equal 'testid'
 
-    it 'Then RouterStore.isWaiting is false', ->
-        RouterStore.isWaiting().should.be.false
+    # it 'Then RouterStore.isWaiting is false', ->
+    #     RouterStore.isWaiting().should.be.false
 
-    it 'Then RouterStore should have some errors', ->
-        should.exist RouterStore.getErrors().get('smtp')
-        error = RouterStore.getErrors().get('smtp')
-        error.message.should.equal 'translated config error smtp'
-        error.should.equal RouterStore.getErrors().get('smtpLogin')
-        error.should.equal RouterStore.getErrors().get('smtpPort')
+    # it 'Then RouterStore should have some errors', ->
+    #     should.exist RouterStore.getErrors().get('smtp')
+    #     error = RouterStore.getErrors().get('smtp')
+    #     error.message.should.equal 'translated config error smtp'
+    #     error.should.equal RouterStore.getErrors().get('smtpLogin')
+    #     error.should.equal RouterStore.getErrors().get('smtpPort')
 
 TEST_ACCOUNT =
     id: 'testid'
```

Comment:
(reminder) We discussed @frankrousseau and I about this file, and this file is not in the right directory; it is functional tests not units.